go.sum

```diff
@@ -32,6 +32,12 @@ github.com/coreos/go-systemd v0.0.0-20181031085051-9002847aa142/go.mod h1:F5haX7
 github.com/coreos/pkg v0.0.0-20160727233714-3ac0863d7acf/go.mod h1:E3G3o1h8I7cfcXa63jLwjI0eiQQMgzzUDFVpN/nH/eA=
 github.com/coreos/pkg v0.0.0-20180928190104-399ea9e2e55f h1:lBNOc5arjvs8E5mO2tbpBpLoyyu8B6e44T7hJy6potg=
 github.com/coreos/pkg v0.0.0-20180928190104-399ea9e2e55f/go.mod h1:E3G3o1h8I7cfcXa63jLwjI0eiQQMgzzUDFVpN/nH/eA=
+github.com/crazycs520/tipb v0.0.0-20200114130449-4ea37911468d h1:eTyHM7/LPx26uBK4Sw4HB6HjeAxhteoHJp8nkFKe+Y0=
+github.com/crazycs520/tipb v0.0.0-20200114130449-4ea37911468d/go.mod h1:J5W0Gvay/AsnE5I7p3JH1RG3HemVvEmp/gSSBCn1NiA=
+github.com/crazycs520/tipb v0.0.0-20200114131211-564e062b68d7 h1:+ZQow7cOpjMrbiBJLMtnSLxxXOrCmRpcbIdY2yUv1jY=
+github.com/crazycs520/tipb v0.0.0-20200114131211-564e062b68d7/go.mod h1:J5W0Gvay/AsnE5I7p3JH1RG3HemVvEmp/gSSBCn1NiA=
+github.com/crazycs520/tipb v0.0.0-20200203123956-7dae12cf6788 h1:GQx+lc3vGId+qGHvnas38CpJ011aKS8drvjOhdEn8fg=
+github.com/crazycs520/tipb v0.0.0-20200203123956-7dae12cf6788/go.mod h1:J5W0Gvay/AsnE5I7p3JH1RG3HemVvEmp/gSSBCn1NiA=
```

Comment:
`make dev` will clean this.

Reply:
need merge `tipd` first and update `go.mod`, https://github.com/pingcap/tipb/pull/169 PTAL